tests/Form/Type/AdminTypeTest.php

```diff
@@ -231,7 +231,7 @@ public function setBar()
                 'sonata_field_description' => $field,
                 'delete' => false, // not needed
                 'property_path' => '[0]', // actual test case
-                'by_reference' => false,
```

Comment:
Maybe we should add a test case using "by_reference" in order to confirm this option is not interfering with the new option.